Client/Frontend/Browser/Onboarding/OnboardingRewardsAgreementViewController.swift

```diff
@@ -0,0 +1,28 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+class OnboardingRewardsAgreementViewController: OnboardingViewController {
+    private var contentView: View {
+        return view as! View // swiftlint:disable:this force_cast
+    }
+    
+    override func loadView() {
+        view = View(theme: theme, themeColour: themeColour)
```

Comment:
I would sync up with @iccub on the way we're supposed to do theming for onboarding